src/H5D.c

```diff
@@ -1075,6 +1119,84 @@ H5Dread_async(const char *app_file, const char *app_func, unsigned app_line, hid
     FUNC_LEAVE_API(ret_value)
 } /* end H5Dread_async() */
 
+/*-------------------------------------------------------------------------
+ * Function:	H5Dread_multi
+ *
+ * Purpose:	Multi-version of H5Dread(), which reads selections from
+ *              multiple datasets from a file into application memory BUFS.
+ *
+ * Return:	Non-negative on success/Negative on failure
+ *
+ * Programmer:	Jonathan Kim Nov, 2013
+ *
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5Dread_multi(size_t count, hid_t dset_id[], hid_t mem_type_id[], hid_t mem_space_id[], hid_t file_space_id[],
+              hid_t dxpl_id, void *buf[] /*out*/)
+{
+    herr_t ret_value = SUCCEED; /* Return value */
+
+    FUNC_ENTER_API(FAIL)
+    H5TRACE7("e", "z*i*i*i*iix", count, dset_id, mem_type_id, mem_space_id, file_space_id, dxpl_id, buf);
+
+    if (count <= 0)
```

Comment:
Should change this to just `if (count == 0)`. Otherwise it'll generate compiler warnings

Reply:
Fixed, thanks